src/graph/sgraph.ts

```diff
@@ -125,7 +125,7 @@ export interface SEdgeSchema extends SModelElementSchema {
  * each of which can be either a node or a port. The source and target elements are referenced via their
  * ids and can be resolved with the index stored in the root element.
  */
-export class SEdge extends SChildElement implements Fadeable, Selectable, Routable, Hoverable {
+export class SEdge extends SChildElement implements Fadeable, Selectable, Routable, Hoverable, BoundsAware {
```

Comment:
Can this lead to other side-effects? The BoundsAware interface is used in several places.